tests/tracking/test_fluent.py

```diff
@@ -681,3 +683,105 @@ def test_delete_tag():
     with pytest.raises(MlflowException):
         mlflow.delete_tag("b")
     mlflow.end_run()
+
+
+@pytest.fixture
+def library_to_mlflow_module():
+    return {
+        "tensorflow": "tensorflow",
+        "keras": "keras",
+        "fastai": "fastai",
+        "pyspark": "spark",
+        "sklearn": "sklearn",
+        "xgboost": "xgboost",
+        "lightgbm": "lightgbm",
+        "mxnet.gluon": "gluon",
+    }
+
+
+@pytest.fixture
+def reset_global_states(library_to_mlflow_module):
```

Comment:
Can we auto-use this fixture so that we don't have to pass it into each test case? See https://docs.pytest.org/en/stable/fixture.html#autouse-fixtures-xunit-setup-on-steroids